en-US/win10/SupportedInterfaces.md

```diff
@@ -41,7 +41,7 @@ Windows 10 IoT Core supports a variety of peripheral interfaces and protocols. H
 *[HID]: Human Interface Device
 
 [1]: {{site.baseurl}}/{{page.lang}}/win10/AllJoyn.htm "AllJoyn Connectivity"
-[2]: http://www.oricothailand.com/2011/index.php?module=main&pages=Product%20info&tp=1&product=560 "ORICO BTA-403 Mini Bluetooth 4.0 USB Dongle"
+[2]: http://www.amazon.com/ORICO-BTA-403-Bluetooth-Adapter-Compatible/dp/B00ESBRTMO/ref=sr_1_7?ie=UTF8&qid=1436917745&sr=8-7&keywords=bluetooth+4.0+orico "ORICO BTA-403 Low Energy Bluetooth 4.0 Adapter"
```

Comment:
Amazon short links are likely better, as this references a specific search criteria and query id. How about http://amzn.com/B00ESBCT56